src/variable.cpp

```diff
@@ -895,11 +917,15 @@ void VariableSlice::setUnit(const Unit &unit) {
 
 template <class T>
 const VariableView<const T> ConstVariableSlice::cast() const {
```

Comment:
Minor point: should you perhaps be using `remove_const` on T prior to doing anything with it?

Reply:
I don't think so: `cast` is currently only used internally in `get`, `T` is obtained from the tag. If someone defines a tag that contains const values then this should also handle T as const.